css-layout-classes/styles.css

```diff
@@ -0,0 +1,32 @@
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+
+  /* border: 5px solid black; */
+}
+
+.row {
+  display: flex;
+  flex-wrap: nowrap;
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+  width: 100%;
```

Comment:
you don't want to set width and flex basis in the same ruleset, as you will end up with undesired effects. The best way to set the images to 100% width is either to select all images in the CSS and set their width to 100%
```css
img {
width: 100%
}
```

or to create an image class and set the value to 100%, then add the class to the images
```css
.images {
width: 100%
}
```